packages/portalnetwork/package.json

```diff
@@ -54,10 +52,17 @@
     "@ethereumjs/common": "^3.0.0",
     "@ethereumjs/tx": "^4.0.0",
     "@ethereumjs/util": "^8.0.0",
+    "@libp2p/bootstrap": "6.0.0",
```

Comment:
Why are all these now devDependencies?  Don't we need them installed in order to use the waku/libp2p stuff?

Reply:
Moved them all up to dependencies.  I'll admit that my grasp of the difference between dependencies and devDependencies is a bit fuzzy...